test/python/test_modeltester.py

```diff
@@ -35,8 +36,18 @@ class TestModelTester(NgraphTest):
     def test_MLP(self):
         cwd = os.getcwd()
         os.chdir('../model_level_tests/')
+        grappler = ngraph_bridge.is_grappler_enabled()
+        varopts = ngraph_bridge.are_variables_enabled()
+        if grappler:
+            if varopts:
+                assert False, "Varopts and grappler does not build together right now"
```

Comment:
Q/Todo: Is this if condition still valid once we merge grappler + varopts

Reply:
Once varopts and grappler are merged, we should have a CI pipeline for that. then this will fail this assert and remind us to enable this then.